.retest/recheck.ignore.js

```diff
@@ -12,7 +12,7 @@ function contains(array, key) {
 }
 
 function shouldIgnoreAttributeDifference(element, diff) {
```

Comment:
shouldn't we change the function name to "matches"?

Reply:
Both works. But yes, we should change this on `master`. Would mind opening a PR?